ERP/resources/views/product-manager.blade.php

```diff
@@ -0,0 +1,329 @@
+            size: "big",
+            color: "red",
+            finishes: "good",
+            grade: "A",
+            quantity: 10
+        },
+        {
+            id:2,
+            type: "street",
+            size: "small",
+            color: "blue",
+            finishes: "bad",
+            grade: "C",
+            quantity: 1
+        },
+        {
+            id:3,
+            type: "street",
+            size: "medium",
+            color: "yellow",
+            finishes: "very good",
+            grade: "A+",
+            quantity: 12
+        },
+        {
+            id:4,
+            type: "mountain",
+            size: "big",
+            color: "red",
+            finishes: "good",
+            grade: "A",
+            quantity: 10
+        },
+        {
+            id:5,
+            type: "mountain",
+            size: "big",
+            color: "red",
+            finishes: "good",
+            grade: "A",
+            quantity: 10
+        },
+        {
+            id:6,
+            type: "mountain",
+            size: "big",
+            color: "red",
+            finishes: "good",
+            grade: "A",
+            quantity: 10
+        },
+    ]
+
+    var bicyleList = new List('bicycles', bicycleOptions, bicycles);
+</script>
+
+<!-- Java script for editing parts -->
+<script>
+    function editParts(id) {  //Java script fucntion for edit parts
+        $('#part_modal').modal('show');
+        alert('edit bicyle id ' + id);
+    }
+
+    partsOptions = { // Parts option from the table column.
+        valueNames: ["type", "color", "quantity"],
+        item: function(item) {
+            return `
+        <tr>
+            <td class="type"></td>
+            <td class="color"></td>
+            <td class="quantity"></td>
+            <td>
+                <button class="btn btn-primary" onClick="editParts(${item.id})">
+                Edit
+                </button>
+                <button class="btn btn-danger" onClick="alert('delete id ' + ${item.id})">
+                Delete
+                </button>
+            </td>
+        </tr>
+        `
+        },
+        pagination: true,
+        page: 5
+    }
+// Variable array should be  data from database.
+    const parts = [{
+            id: 1,
+            type: "mountain",
+            color: "red",
+            quantity: 10
+        },
+        {
+            id: 2,
+            type: "mountain",
+            color: "red",
+            quantity: 10
+        },
+        {
+            id: 3,
+            type: "mountain",
+            color: "red",
+            quantity: 10
+        },
+        {
+            id: 4,
+            type: "street",
+            color: "green",
+            quantity: 20
+        },
+        {
+            id: 5,
+            type: "mountain",
+            color: "red",
+            quantity: 10
+        },
+
+    ]
+
+    var bicyleList = new List('parts', partsOptions, parts);
+</script>
```

Comment:
@abirshah you did a really good job, but this is still a PHP framework we are working with, so the functions will need to be in the controllers section with PHP code. Otherwise, very good job

Reply:
@DGovi that's basically dynamically adding button for every row. Every time, we have new record in the table, we need to have Edit and Delete. Yes all the controller will be in php laravel. Html doesn't have ability to add  Edit and Delete button dynamically. As far as I know php works with all of them nicely. You probably know better than me because you are php guy.